robot_code/system_demo/system_demo.ino

```diff
@@ -42,7 +43,7 @@ void loop(void)
     humanhand.updateSensors();
     robothand.updateSensors();
 
-    fingerStatus = humanhand.getFingerStatus();
+    pointerStatus, thumbStatus = humanhand.getFingerStatus();
```

Comment:
Doesn't `humanhand.getFingerStatus()` set these attributes within the class? Also, I don't think `humanhand.getFingerStatus()` outputs any values.

Reply:
nvm you gucci